src/Controller/BookingsController.php

```diff
@@ -0,0 +1,75 @@
+<?php
+
+namespace RebelCode\EddBookings\RestApi\Controller;
+
+use Dhii\Storage\Resource\SelectCapableInterface;
+use RebelCode\EddBookings\RestApi\Resource\BookingResource;
+use RebelCode\EddBookings\RestApi\Resource\ResourceFactoryInterface;
+
+/**
+ * The API controller for bookings.
+ *
+ * @since [*next-version*]
+ */
+class BookingsController extends AbstractBaseCqrsController
+{
+    /**
+     * Constructor.
+     *
+     * @since [*next-version*]
+     *
+     * @param ResourceFactoryInterface $resourceFactory The resource factory.
+     * @param SelectCapableInterface   $selectRm        The bookings resource model.
+     * @param object                   $exprBuilder     The expression builder.
+     */
+    public function __construct(
+        ResourceFactoryInterface $resourceFactory,
+        SelectCapableInterface $selectRm,
+        $exprBuilder
+    ) {
+        $this->resourceFactory = $resourceFactory;
+        $this->selectRm        = $selectRm;
+        $this->exprBuilder     = $exprBuilder;
+    }
+
+    /**
+     * {@inheritdoc}
+     *
+     * @since [*next-version*]
+     */
+    protected function _getParamsInfo()
```

Comment:
When you start implementing `put()`, `post()`, and `delete()`, you will most likely have to rename this to something like `_getParamsInfoGet()`.